### PR TITLE
Release 2.6

### PR DESCRIPTION
**Pull request by JanneKiiskila**
Pelion Edge 2.6 release related changes (w LmP v85).
* Use `df` for partition size checks instead of `df -h`.
* Update node and npm versions to fix npm audit issues.
* Drop HW variant tests.
* Contents of `wip` branch also merged in, see #16 
- LmP v89 support
- Remove EU configs
- Easier UX, one script to do it all (`scripts/perts.sh -a <access-key>`).
- GitHub action for shellchecking scripts.
- Improved support for VM-type machines running snap.
- Improved logging for pods.
- Automated network interface name handling for wired interface if name `default` is used.
- README.md updated.
- Change i.mx8 sysroot device name.
- NPM and Node.JS version upgrades.
    - This also changes the way `localhost` is resolved.
    - It gets resolved into `::1` as an IPv6 address instead of `127.0.0.1`, so the edge-core test has to use the hard-coded IPv4 address 127.0.0.1 instead of `localhost`.

### CODE REVIEW

**Comment by marcuschangarm** [comment]
The commit log is different than the one we've been reviewing, https://github.com/PelionIoT/pelion-edge-ready-test-suite/pull/12, what happened?

**Comment by JanneKiiskila** [comment]
I've been tweaking the history a bit, the amount of commits etc. was becoming messy. I'm tempted to do a bit more of `git rebase -i` magic and squash a few ones out still even on this one.

**Comment by marcuschangarm** [comment]
You should have let us approve it before tidying up the commit log! 😄

**Review by marcuschangarm — APPROVED** [review]

**Review by petedyer11 — APPROVED** [review]
LGTM

**Comment by jenia81** [comment]
Can we squash the commits a bit?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
-# Pelion Edge ready test suite
+# Izuma Edge ready test suite
 
-For documentation about the Pelion Edge ready test suite, please see the [Pelion Edge documentation](https://developer.pelion.com/docs/device-management-edge/latest/testing/pelion-edge-ready-test-suite.html).
+For a more thorough documentation about the Izuma Edge ready test suite, please see the [Izuma Edge documentation](https://developer.izumanetworks.com/docs/device-management-edge/latest/testing/izuma-edge-ready-test-suite.html).
+
+# Quick start
+
+Run the [`script/perts.sh -a <access-key> -c <config-file-template>`](scripts/perts.sh).
+- [`test-configs`](test-configs) -folder has multiple template scripts.
```
Comment (jenia81) [inline README.md]:
```suggestion
- [`test-configs`](test-configs) - folder has multiple template scripts.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+- The script will create a `test-config.json` -file which it uses as the configuration file running the test.
```
Comment (jenia81) [inline README.md]:
```suggestion
- The script will create a `test-config.json` file which is used as the configuration file for the test.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+- It will automatically populate the device ID and account ID based on the information it gets via `curl localhost:<port>/status`.
+
+**NOTE! KaaS tests require that the account has Edge KaaS features enabled. By default KaaS features are not enabled. Contact [Iazuma Networks](https://www.izumanetworks.com/) to enable it.**
```
Comment (jenia81) [inline README.md]:
```suggestion
**NOTE! KaaS tests require that the account has Edge KaaS features enabled. By default, KaaS features are not enabled. Contact [Izuma Networks](https://www.izumanetworks.com/) to enable it.**
```

File: scripts/swap_td_config.sh [hunk 4]
```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
 # ----------------------------------------------------------------------------
 # Copyright (c) 2020-2021, Pelion and affiliates.
```
Comment (jenia81) [inline scripts/swap_td_config.sh:3]: Izuma

File: test-cases/l2_program_tests/program_and_services.js [hunk 5]
```diff
@@ -71,47 +72,29 @@ describe('[Level 2] ServicesAndProgramExistanceTests', () => {
...
+                true,
+                'SSL version not valid'
+              )
+              done()
+            }
           }
-        }
-      )
-    })
-    it('Should return true if valid openssh version is present', function (done) {
```
Comment (jenia81) [inline test-cases/l2_program_tests/program_and_services.js:98]: why is the open ssh part was removed?

Comment (JanneKiiskila) [reply test-cases/l2_program_tests/program_and_services.js:98]: We don't need openssh at all for anythig - the version of it really depends fully on what OS version you happen to use.

**Review by jenia81 — APPROVED** [review]
